src/app/payment/paymentMethod/PaypalCommercePaymentMethod.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useCallback, useContext, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
```

Comment:
I believe `Omit` is available in TypeScript by default, so you don't need to pull it in through `utility-types`